roofit/roofitcore/src/TestStatistics/RooBinnedL.cxx

```diff
@@ -87,6 +95,9 @@ RooBinnedL::evaluatePartition(Section bins, std::size_t /*components_begin*/, st
    // expensive than that, we tolerate the additional cost...
    ROOT::Math::KahanSum<double> result;
 
+   // Do not reevaluate likelihood if parameters have not changed
+   if (!_paramTracker->hasChanged(true) & (_cachedResult != 0)) return _cachedResult;
```

Comment:
```suggestion
   if (!pdf_->isValueDirty()) return _cachedResult;
```
Same comment as before: I'm not sure the change tracker is necessary.

Reply:
Resolved, as discussed below.